src/rvsdg/egglog_optimizer/subst.egg

```diff
@@ -91,3 +91,11 @@
   ( 
     (union (SubstVecVecOperandHelper (VVO vec) x v i) (VVO vec))
   ) :ruleset subst)
+ 
+
+;; ####################################
+;; implementation of PassThroughArguments
+;; Creates a vec of arguments
+;; (vec-of (Arg 0) (Arg 1) ...) with length i
+(function PassThroughArguments (i64) VecOperand :unextractable)
```

Comment:
Can you get rid of this stub by reordering the include_strs, or is there a circular dependency? (If there is a circular dependency please document it.)

Reply:
Moved to `util.egg` good catch